portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesInterests.testcase

```diff
@@ -91,19 +55,21 @@ definition {
 
 			ACCards.viewCards(cardTitle = "Interests");
 
-			ACTimeFilter.assertCurrentTimeFilterRange(timeFilterRange = "Last 30 days");
+			ACTimeFilter.assertCurrentTimeFilterRange(
+				cardName = "Interests",
```

Comment:
Hey @joshitagaki, I think the correct name of the card would be Top Interests, no? Or does it work using only Interests?

Reply:
Ohhh yeah, I thought it was Top Interests, but that's in segments. Thanks Josh